docs/technical/resolve-merge-conflicts.md

```diff
@@ -111,15 +148,21 @@ And it's ready to merge:
 
 ## Dependency updates
 
-When dealing with dependency and lock file updates there are multiple use cases to consider, however as a baseline, the OpenSauced triage team will not prioritize parallel main features as seen in the roadmap.
+When dealing with dependency and lock file updates there are multiple use cases to consider, however, as a baseline, the OpenSauced triage team will not prioritize parallel main features as seen in the roadmap.
```

Comment:
I'm not sure what happened here with deleting and adding the paragraph back in. If documentation is ok, try to make sure you leave it as it was before. It's ok in this case, but for future reference, it makes code review easier.